mlflow/store/file_store.py

```diff
@@ -424,9 +424,10 @@ def _get_metric_from_file(parent_path, metric_name):
     def get_all_metrics(self, run_uuid):
         _validate_run_id(run_uuid)
         parent_path, metric_files = self._get_run_files(run_uuid, "metric")
-        metrics = []
+        metrics = {}
```

Comment:
These functions are all called into by `get_run` (which is also their only caller outside of tests)